cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -156,3 +156,20 @@ Feature: Users can authneticate with OIDC authenticator
     """
     Errors::Authentication::AuthnOidc::AdminAuthenticationDenied
     """
+
+  Scenario: Performance test
+    Given I get authorization code for username "alice" and password "alice"
+    And I fetch an ID Token
+    When I authenticate "1000" times in "10" threads via OIDC with id token
+    Then The "max" response time should be less than "1" seconds
+    And The "avg" response time should be less than "0.25" seconds
+
+  Scenario: Load test
+    Given I get authorization code for username "alice" and password "alice"
+    And I fetch an ID Token
+    And I save my place in the log file
+    When I authenticate "2000" times in "20" threads via OIDC with id token
+    Then The following appears in the log after my savepoint:
+    """
+    Completed 200 OK
```

Comment:
i'm not sure this is enough. maybe add also the line of sending the request. should be something like `POST .../authn-oidc/authenticate`

Reply:
Checking `POST .../authn-oidc/authenticate` isn't enough to know that the request was successful